src/cogent3/__init__.py

```diff
@@ -735,11 +732,15 @@ def load_table(
     )
 
 
+@c3warn.deprecated_args(
+    version="2024.12", reason="argument has no effect", discontinued="name_nodes"
+)
 def make_tree(
     treestring=None,
     tip_names=None,
     format=None,
     underscore_unmunge=False,
+    name_nodes=False,
```

Comment:
delete the argument and remove it from the docstring, make sure it's not used in any tests too